.github/comment.yml

```diff
@@ -4,14 +4,13 @@
     repoAllowList:
       - Azure/azure-rest-api-specs-pr
     variables:
-      openapiHub: https://portal.azure-devex-tools.com
       to: Azure/azure-rest-api-specs/main
     label: Approved-OkToMerge
     onLabeledComments: >-
       Hi @${PRAuthor}! Your PR is approved. Congratulations. :partying_face::rocket: </br>
       If this PR is targeting `main` branch, then it cannot be merged, as `azure-rest-api-specs-pr` repo `main` branch 
       is mirrored from `azure-rest-api-specs` `main` branch.<br/>
-      If you want to publish the PR to the public repo (`Azure/azure-rest-api-specs`) `main` branch,
+      If you want to move your changes to the public repo (`Azure/azure-rest-api-specs`) `main` branch,
```

Comment:
I think `publish the PR`, as it was, fits better here. The goal is to publish the spec to public - the fact PR is getting "moved" is an implementation details. If we make it say `move your changes` it may be less clear to people that by "moving the changes" they are in fact publishing the changes to the public.

Reply:
changed to publish